js-intro.js

```diff
@@ -11,51 +11,118 @@ var temp1 = 35
 var temp2 = 350
 var temp3 = 212
 
-
-
+// Boiling point = 212
+const boilingTemp = (temp) => {
+    // Checks to see if input is of number data type
+    if (typeof temp === "number") {
+        if (temp === 212) {
+            return `${temp} is at boiling point.`;
+        } else if (temp < 212) {
+            return `${temp} is below boiling point.`;
+        } else if (temp > 212) {
+            return `${temp} is above boiling point.`;
+        }
+      // If not of number data type, message appears         
+    } else {
+        return 'Please enter a number and try again.';
+    }
+}
 
 // --------------------2) Create a function that multiplies each number in the array by 5 using a for loop.
 // Use the test variable provided below. Expected outcome: [15, 35, 0, 30, -45]
 
-var myNumbers1 = [3, 7, 0, 6, -9]
-
+var myNumbers1 = [3, 7, 0, 6, -9];
 
+// Function to run array items * 5 using for loop
+const timesFive = (arr) => {
+    // Declare empty array to store calculated results
+    let calculatedArray = [];
+    // for loop to stop running once variable i is one less than the length of the inputed array
+    for (var i = 0; i < arr.length; i++) {
+        calculatedArray.push(arr[i] * 5);
+    }
+    return calculatedArray;
+}
 
 
 // --------------------3) Create a function that multiplies each number in the array by 5 using map.
 // Use the test variable provided below. Expected outcome: [40, -35, 0, 30, 10]
 
 var myNumbers2 = [8, -7, 0, 6, 2]
 
-
-
+// Function to run array item * 5 using map
+const timesFiveV2 = (arr) => {
+    let calculatedArray = [];
+    calculatedArray = arr.map(el => el * 5);
+    return calculatedArray;
+}
 
 // --------------------4) Create a function that removes all the vowels from a string.
 // Use the test variables provided below. Expected output: "HyThrLrnStdnt" "LvJvScrpt"
 
-var stringWithVowels1 = "HeyThereLearnStudent"
-var stringWithVowels2 = "ILoveJavaScript"
-
-
-
-
+var stringWithVowels1 = 'HeyThereLearnStudent'
+var stringWithVowels2 = 'ILoveJavaScript'
+
+// Function that removes all vowels from any string input
+const removeVowels = (str) => {
+    const vowels = ['a', 'i', 'e', 'o', 'u'];
+    let splitStr;
+    let newStrArr;
+    let newStr;
+    
+    // Letters are split and converted into an array
+    splitStr = str.split('');
+    // Condition pushes the splitStr's element into newStrArr if that element is not found within the vowels array
+    // toLowerCase() is used so that all comparisons are checked correctly since the strings within the vowels array are also lowercase
+    newStrArr = splitStr.filter(el => vowels.indexOf(el.toLowerCase()) === -1);
+    // Join letters together; join() also converts the array into a string
+    newStr = newStrArr.join('');
+    return newStr;
+}
```

Comment:
I love the line of logic here! But you can actually shorten this code to just two lines.

`const vowels = ['a', 'i', 'e', 'o', 'u'];`
`return str.split('').filter(el => vowels.indexOf(el.toLowerCase()) === -1).join('');`

Your solution works completely and this is just a suggestion. If the way you have it written above helps you with your work flow, that is totally fine.